### PR TITLE
Whitelist files in package

### PR DESCRIPTION
**Pull request by eliangcs**
Same stuff as https://github.com/zapier/zapier-platform-cli/pull/330.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,10 @@
   "license": "UNLICENSED",
   "main": "index.js",
   "typings": "index.d.ts",
+  "files": [
+      "/include/*.js",
+      "/src/**/*.js"
```
Comment (xavdid) [inline package.json]: Needs the following:

* index.js
* index.d.ts

File: package.json [hunk 2]
```diff
@@ -8,6 +8,10 @@
   "license": "UNLICENSED",
   "main": "index.js",
   "typings": "index.d.ts",
+  "files": [
+      "/include/*.js",
```
Comment (xavdid) [inline package.json]: like `cli`, I think this can be `/include/` and `/src/`, though double check the output of `npm pack` to verify that.

**Review by xavdid — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (eliangcs) [reply package.json]: Thanks! Added the two files in c86230c.

NOTE: `index.js` is automatically included as it's specified as `main`, but I added it to `files` anyway just to be more explicit.

File: package.json (hunk 2 above)
Comment (eliangcs) [reply package.json]: Fixed in c86230c.

**Comment by eliangcs** [comment]
@xavdid updated the PR. Thanks!

**Comment by xavdid** [comment]
cool. didn't explicitly test, but it looks good!

**Review by xavdid — APPROVED** [review]

**Pull request merged**